src/webots/nodes/WbElevationGrid.cpp

```diff
@@ -57,6 +57,12 @@ void WbElevationGrid::init() {
 
 WbElevationGrid::WbElevationGrid(WbTokenizer *tokenizer) : WbGeometry("ElevationGrid", tokenizer) {
   init();
+
+  // avoid warning
```

Comment:
I am afraid this is not the right way to fix the problem because we don't want to silently override the value set by the user...

Reply:
True but I was not seeing why an user would like to create such an element and keep those field values to 0.

Reply:
I agree there is no point having these values set to 0, but overriding silently values set in a user file (world or proto) is not a good practice in general.